### ISSUE TITLE
Add normalization layer

### ISSUE THREAD

**Issue by dguest** [lwtnn/lwtnn#28]
Having a "normalization" layer could be useful in a few ways:
 - We could replace the `InputPreprocessor` with a normalization layer
 - More importantly, we could support "batch" normalization without the batches

Keras stores the running mean and standard deviation from batch normalization, so converting this into layer information should be relatively easy.

**Comment by Marie89** [comment]
Given the improvements we see using it for b-tagging (see [FTag Algo meeting update on DL1](https://indico.cern.ch/event/571869/#preview:2091811)), it would be good to add it to rel21 while it's still possible. Meaning within the next week. 
Is this timescale feasible?

**Comment by jwsmithers** [comment]
Hi Marie, 
This can potentially happen by, say, Wednesday? Would that be soon enough?

**Comment by Marie89** [comment]
Hey Josh,
if this means that it's validated on a simple architecture, implemented in rel21 and the [Keras2JSON converter](https://github.com/lwtnn/lwtnn/blob/master/converters/keras2json.py) is adapted correctly by say Wednesday noon, I guess yes. FYI, I'm currently using Keras version 1.1.1 and w.r.t. BN there were a few changes in the previous versions, so in case people are using older versions and would like to benefit from using it in Athena. I can run a last validation check using a NN that uses BN on Wednesday afternoon if it's working for DL1 - just as a very last validation check.
But just to make sure, I'd also ask Richard and Manu on the timeline they have in mind for closing the release to changes.

**Comment by jwsmithers** [comment]
The converter is done, it's just implementing the layer now and validating which I suspect will be the hardest part. I'll probably be asking Dan for help.
I'll shoot for having this done by Wednesday, but this will be very tight.

**Comment by dguest** [comment]
This should be complete as of pull request #30
